app/src/main/java/org/squidfish/tuplenback/views/TupleNBackApp.kt

```diff
@@ -105,12 +140,12 @@ fun TupleNBackApp(
                         navController.navigate(ScreenDestination.GameSelectionScreen)
                     },
                     onPlayRecent = {
-                        if (gameModel.game == null) {
+                        if (gameModel.level.value == null) {
                             Log.w(TAG, "Cannot play recent: No game played previously")
                             return@MainMenuScreen
                         }
 
-                        navController.navigate(ScreenDestination.GameScreen(game = null))
+                        navController.navigate(ScreenDestination.GameScreen(null, 0))
```

Comment:
When using constants, can we keep the labels for clarity? I don't know why detekt is not complaining but I am a little bit, I had to check line 111 to see what `0` means